set-up.R

```diff
@@ -16,25 +17,133 @@ pkgs <- c(
   "stplanr", # Sustainable transport planning with R
   "tidyr", # tidies up your data!
   "readr", # reads your data fast
-  "knitr", # for knitting it all together
-  "geojsonio",
-  "rmapshaper" # To simplify rnet
-  )
-# Which packages do we require?
-# lapply(pkgs, library, character.only = T)
+  "knitr" # for knitting it all together
+)
+
+# Github packages to install.
+ghpkgs <- c(
+  "ateucher/rmapshaper"
+)
+
+# Package names for GitHub
+ghpkgns <- c(
+  "rmapshaper"
+)
+
+# Ensure packages are up-to-date.
+cat("It is strongly recommended that you update your installed packages.\n");
+update.packages()
```

Comment:
This is pretty dramatic!  I'm don't really want to forcibly update users packages without asking them first...
@Robinlovelace any thoughts on this?  I don't know how compatible this approach is with the description file https://github.com/npct/pct-load/commit/bb3b87787564597947614b2c3863aff36e853e6f